aws_lambda_powertools/utilities/batch/base.py

```diff
@@ -72,7 +110,7 @@ def __call__(self, records: List[Any], handler: Callable):
         self.handler = handler
         return self
 
-    def success_handler(self, record: Any, result: Any):
+    def success_handler(self, record, result: Any) -> SuccessCallback:
```

Comment:
Would `SuccessResponse` be a more accurate name?

Reply:
YES! Good catch -- will change that